omod/src/main/java/org/openmrs/module/registrationapp/fragment/controller/RegisterPatientFragmentController.java

```diff
@@ -271,13 +273,14 @@ public FragmentActionResult submit(UiSessionContext sessionContext, @RequestPara
         if (registrationEncounter != null) {
             redirectUrl = redirectUrl.replaceAll("\\{\\{encounterId\\}\\}", registrationEncounter.getId().toString());
         }
-
-        return new SuccessResult(redirectUrl);
+        SimpleObject result = SimpleObject.create("redirectUrl", redirectUrl, "uuid", patient.getUuid());
+        ObjectResult response = new ObjectResult(result);
+        return response;
```

Comment:
What does this change do?

Reply:
I needed to get the patient uuid in the response since the saving of the image is done via the  ajax success call back [here](https://github.com/openmrs/openmrs-module-registrationapp/pull/71/files#diff-a8dba45e0d7668aff40327eefc1c1ff3R244)